src/generated/randr.rs

```diff
@@ -4356,38 +4524,255 @@ impl TryParse for NotifyData {
     }
 }
 impl From<CrtcChange> for NotifyData {
-    fn from(value: CrtcChange) -> Self {
-        Self(value.serialize())
+    fn from(cc: CrtcChange) -> Self {
+        let cc_bytes = cc.serialize();
+        let value = [
+            cc_bytes[0],
+            cc_bytes[1],
+            cc_bytes[2],
+            cc_bytes[3],
+            cc_bytes[4],
+            cc_bytes[5],
+            cc_bytes[6],
+            cc_bytes[7],
+            cc_bytes[8],
+            cc_bytes[9],
+            cc_bytes[10],
+            cc_bytes[11],
+            cc_bytes[12],
+            cc_bytes[13],
+            cc_bytes[14],
+            cc_bytes[15],
+            cc_bytes[16],
+            cc_bytes[17],
+            cc_bytes[18],
+            cc_bytes[19],
+            cc_bytes[20],
+            cc_bytes[21],
+            cc_bytes[22],
+            cc_bytes[23],
+            cc_bytes[24],
+            cc_bytes[25],
+            cc_bytes[26],
+            cc_bytes[27],
+        ];
+        Self(value)
```

Comment:
I think this is the same bug/difference as I already found on xproto's `ClientMessageData`. So, this comment is just a marker for future-me to check if this difference went away.

Reply:
Fixed